src/libsyntax/print/pprust.rs

```diff
@@ -179,7 +179,7 @@ pub fn literal_to_string(lit: token::Lit) -> String {
         token::ByteStrRaw(n) => format!("br{delim}\"{string}\"{delim}",
                                         delim="#".repeat(n as usize),
                                         string=symbol),
-        token::Err           => format!("'{}'", symbol),
+        token::Err           => format!("{}", symbol),
```

Comment:
```suggestion
        token::Err           => symbol.to_string(),
```

to match the `Integer` branch above